src/models.rs

```diff
@@ -25,6 +25,7 @@ pub struct DeletedMessage {
 pub struct Room {
     pub id: String,
     pub name: String,
+    pub pinned_message_id: i64,
```

Comment:
I think should be an `Opt<i64>`

Reply:
... and I believe that:
```
    #[serde(skip_serializing_if = "Option::is_none")]
    pinned_message_id: Option<i64>,
```
should make it be omitted when set to `None`.  (Assuming you want that; without it the default behaviour would still include with a `null` value).